src/test/scala/org/kibanaLoadTest/simulation/BaseSimulation.scala

```diff
@@ -35,19 +38,23 @@ class BaseSimulation extends Simulation {
 
     // saving deployment info to target/lastDeployment.txt"
     if (appConfig.deploymentId.isDefined) {
+      logger.info(s"Getting Kibana status info")
+      val response = new HttpHelper(appConfig).getStatus()
       val meta = Map(
         "deploymentId" -> appConfig.deploymentId.get,
         "baseUrl" -> appConfig.baseUrl,
-        "version" -> appConfig.buildVersion
+        "version" -> appConfig.buildVersion,
+        "buildHash" -> response.extract[String]('version / 'build_hash),
+        "buildNumber" -> response
+          .extract[Int]('version / 'build_number)
```

Comment:
This `[Int]` is a type-cast?

Reply:
yes, when you read json path with lenses you need to explicitly set value type